QubitServer/src/main/java/org/labrad/qubits/config/AdcBaseConfig.scala

```diff
@@ -0,0 +1,41 @@
+package org.labrad.qubits.config
+
+import java.util.Map
+
+import org.labrad.data.Request
+
+abstract class AdcBaseConfig(channelName: String, builProperties: Map[String, Long]) {
+
+  /**
+   * number of clock cycles to delay
+   */
+  protected var startDelay: Int = -1
```

Comment:
so, when I take out the member variable itself, things fail to compile because the `reconcile` method checks that it is > -1 or else fails with an error that the delay was not set. But the getter is never called; the only reason this check is passing now is that `startDelay` is getting initialized to the default value of 0, which is obviously greater than -1...